### ISSUE TITLE
Support Classes & Types in ExportedTypes

### ISSUE THREAD

**Issue by leannepepper** [souporserious/renoun#68]
The `ExportedTypes` component works great for function declarations and expressions. However the component doesn't seem to return anything for exported types and classes.

Can this be added in a future release?

**Comment by souporserious** [comment]
Ah, yeah classes and other exports like types aren't included yet, but will be in a future release! I'll update this issue once they are supported.

**Comment by leannepepper** [comment]
I tried the new `APIReference` component which looks like it added support for classes, so thank you for that! However, I'm getting this error when trying to document class properties that are using MobX decorators: 

```
[resolveClassPropertyDeclaration] Property "[$mobx]" could not be resolved. This declaration was either filtered, should be marked as internal, or filed as an issue for support
```

Any ideas if this will be supported?

**Comment by souporserious** [comment]
Thanks for letting me know decorators weren't working! I add support [here](https://github.com/souporserious/renoun/commit/09e4efd8cdf157e3da40c2d7e1f5fa1e66aa4155) and it's released in the latest version [`5.4.0`](https://github.com/souporserious/renoun/releases/tag/renoun%405.4.0). Most types should be covered now, but please feel free to reopen this issue if you run into anything else.
